### PR TITLE
output json as array instead of dictionary

### PR DESCRIPTION
**Pull request by NicklasHugoy**

### CODE REVIEW

File: data_handler.py [hunk 1]
```diff
@@ -47,16 +47,24 @@ def _create_smiley_json(self):
...
+        res = []
 
         for row in list(root):
             new_obj = {col.tag: col.text for col in row}
-            key = row[0].text
-            res[key] = new_obj
+            new_obj['Geo_Lat'] = float(new_obj['Geo_Lat']) if new_obj['Geo_Lat'] != None else "null"
+            new_obj['Geo_Lng'] = float(new_obj['Geo_Lng']) if new_obj['Geo_Lng'] != None else "null"
+            self.convert_control_to_int(new_obj)
+            new_obj['navnelbnr'] = row[0].text
```
Comment (Badgie) [inline data_handler.py]: Can just be removed. Was only there to assign a key to the dict before.

File: data_handler.py [hunk 2]
```diff
@@ -47,16 +47,24 @@ def _create_smiley_json(self):
...
 
-        res = {}
+        res = []
 
         for row in list(root):
             new_obj = {col.tag: col.text for col in row}
-            key = row[0].text
-            res[key] = new_obj
+            new_obj['Geo_Lat'] = float(new_obj['Geo_Lat']) if new_obj['Geo_Lat'] != None else "null"
+            new_obj['Geo_Lng'] = float(new_obj['Geo_Lng']) if new_obj['Geo_Lng'] != None else "null"
```
Comment (Badgie) [inline data_handler.py]: Could be a method alike `convert_control_to_int()`

**Review by Badgie — APPROVED** [review]
:shipit:

File: data_handler.py (hunk 1 above)
Comment (Badgie) [reply data_handler.py]: Oh. refers to line 57 only

**Comment by NicklasHugoy** [comment]
@Badgie 
Jeg har lavet metoder til konvertering af typer.
Tænker at det læses bedre?

**Comment by Badgie** [comment]
Det er smukt.

Also,

JEG ER SÅ STOLT OVER DET LIST UNPACKING DU HAR GANG I

**Pull request merged**